sunbird_service_api_test/src/test/java/org/sunbird/common/util/HttpUtil.java

```diff
@@ -51,22 +60,34 @@ public static boolean doDeleteOperation (String url) {
 		}
 		return deleteResponse;
 	}
-	
-	public static String doGetOperation(String url, Map<String, String> headers) {
-		StringBuffer result = new StringBuffer();
-		try (CloseableHttpClient httpclient = HttpClients.createDefault()) {
-			HttpGet request = new HttpGet(url);
-			request.addHeader("content-type", ConstantKeys.CONTENT_TYPE_APPLICATION_JSON);
-			HttpResponse response = null;
-			response = httpclient.execute(request);
-			BufferedReader rd = new BufferedReader(new InputStreamReader(response.getEntity().getContent()));
-			String line = "";
-			while ((line = rd.readLine()) != null) {
-				result.append(line);
+
+	/**
+	 * Send multipart HTTP post request with form data.
+	 *
+	 * @param httpClient HTTP client to use for sending the request.
+	 * @param config Configuration (e.g. API key) used in sending HTTP request
+	 * @param url HTTP URL to use in the request
+	 * @param formDataFile File path containing each form parameter in a new line in format (key=value)
+	 */
+	public void multipartPost(HttpClientActionBuilder httpClientActionBuilder, TestGlobalProperty config, String url, String formDataFile) {
+		MultiValueMap<String, Object> formData = new LinkedMultiValueMap<>();
+
+		try (Scanner scanner = new Scanner(new File(getClass().getClassLoader().getResource(formDataFile).getFile()))) {
+
+			while (scanner.hasNext()) {
+				String[] param = scanner.nextLine().split("=");
+				if (param.length == 2) {
+					formData.add(param[0], param[1]);
+				}
 			}
-		} catch (Exception e) {
+
+		} catch (IOException e) {
 			e.printStackTrace();
 		}
-		return result.toString();
+
+		httpClientActionBuilder.send().post(url)
+				.contentType("multipart/form-data")
```

Comment:
we can create content type enum in Constant "multipart/form-data"

Reply:
Used MediaType.MULTIPART_FORM_DATA.